lib/pbench/test/unit/server/test_requests.py

```diff
@@ -43,7 +43,7 @@ def test_host_info(client, pytestconfig, caplog, server_config):
         assert response.status_code == 200
         assert response.json.get("message") == expected_message
         for record in caplog.records:
-            assert record.levelname not in ("WARNING", "ERROR", "CRITICAL")
+            assert record.levelname == "INFO"
```

Comment:
You really want the test to fail if someone adds a `DEBUG` level message?

Reply:
What about the other test scenarios?  Shouldn't those ignore `DEBUG` level messages, too?

Reply:
Moved to #2287.